packages/context/src/index.ts

```diff
@@ -33,7 +35,11 @@ export class Services implements ServicesInterface {
         throw e;
       });
     }
-    void this.walletServicesPromise.then(({ blockProcessor }) => blockProcessor.sync());
+    void this.walletServicesPromise.then(({ blockProcessor }) =>
+      this.config.chainId === 'penumbra-1'
+        ? blockProcessor.sync(genesisBlock as JsonObject)
+        : blockProcessor.sync(),
```

Comment:
CI currently failing because `sync` method in https://github.com/penumbra-zone/web/pull/1706# was modified to optionally accept bundled genesis compact block.